Source/WebViewController.swift

```diff
@@ -222,5 +222,10 @@ open class WebViewController: UIViewController {
         let completed = webView.estimatedProgress == 1.0
         progressBar.setProgress(completed ? 0.0 : Float(webView.estimatedProgress), animated: !completed)
         UIApplication.shared.isNetworkActivityIndicatorVisible = !completed
+        webView.evaluateJavaScript("document.title", completionHandler: { (data:Any?, error:Error?) in
```

Comment:
this should be wrapped in `if completed`, right? 

otherwise, this will be executed quite often